<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-ATJLIE4" failures="4" tests="9" name="Test1" time="19.174" errors="0" timestamp="2024-12-29T15:58:22 IST">
  <testcase classname="api.test.Cart_Tests" name="testCartValue" time="11.719"/>
  <testcase classname="api.test.Cart_Tests" name="testCartValue2" time="0.253">
    <failure type="java.lang.AssertionError" message="expected [0] but found [8]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [8]
at api.test.Cart_Tests.testCartValue2(Cart_Tests.java:70)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testCartValue2 -->
  <testcase classname="api.test.Cart_Tests" name="testFlatXAmountOff" time="0.23"/>
  <testcase classname="api.test.Cart_Tests" name="testFlatX_Percentage_Off" time="0.23"/>
  <testcase classname="api.test.Cart_Tests" name="testNegativeCartValue" time="0.151">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &amp;lt;400&amp;gt; but was &amp;lt;200&amp;gt;.
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <400> but was <200>.

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:57)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at api.test.Cart_Tests.testNegativeCartValue(Cart_Tests.java:76)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testNegativeCartValue -->
  <testcase classname="api.test.Cart_Tests" name="testNegativeOfferValue" time="0.398">
    <failure type="java.lang.AssertionError" message="expected [100] but found [90]">
      <![CDATA[java.lang.AssertionError: expected [100] but found [90]
at api.test.Cart_Tests.testNegativeOfferValue(Cart_Tests.java:83)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testNegativeOfferValue -->
  <testcase classname="api.test.Cart_Tests" name="testUpdatedDiscount" time="0.321">
    <failure type="java.lang.AssertionError" message="expected [170] but found [180]">
      <![CDATA[java.lang.AssertionError: expected [170] but found [180]
at api.test.Cart_Tests.testUpdatedDiscount(Cart_Tests.java:93)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- testUpdatedDiscount -->
  <testcase classname="api.test.Cart_Tests" name="testUserWithNoRestauId" time="1.665"/>
  <testcase classname="api.test.Cart_Tests" name="testMockerUserCreation" time="0.842"/>
</testsuite> <!-- Test1 -->
